README.md

```diff
@@ -54,4 +54,9 @@ Finally, if you are sure no one else had the issue, follow the [ISSUE_TEMPLATE](
 
 ### Tip
 
-Sometimes to program ESP32 via serial you must keep GPIO0 LOW during the programming process
+1 > Sometimes to program ESP32 via serial you must keep GPIO0 LOW during the programming process
+
+2 > On some ESP32 development boards, When you trying to upload a new sketch, the Arduino IDE gives you a fatal error. ( like A fatal error occurred: Failed to connect to ESP32: Timed out waiting for packet header) This means your ESP32 board does not goes to automatically into flashing/uploading mode. 
```

Comment:
2 > In some ESP32 development boards, when trying to upload a new sketch, the Arduino IDE gives you a fatal error (i.e *A fatal error occurred: Failed to connect to ESP32: Timed out waiting for packet header*).
This means your ESP32 board is not automatically in the flashing/uploading mode.

Reply:
Good to know, Some brief comments. All changes are added in README.md.
Thank You.